nextflow.config

```
@@ -62,6 +62,12 @@ params {
     // General module configuration
     multiqc.config         = "$projectDir/configs/multiqc_conf.yaml"
     multiqc_assembly_report_config = "$projectDir/configs/multiqc_assembly_report_config.yml"
+
+    // define HiC mapping pipeline [ "bwa", "bwa-mem2", "minimap2", "strobealign", "" ]
+    hic_mapper             = 'bwa-mem2' 
```

```suggestion
    hic_aligner             = 'bwa-mem2' 
```
I would probably use the term aligner rather than mapper, but that's not a huge deal